prow/cmd/mkpj/main.go

```diff
@@ -79,6 +170,10 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Error loading config.")
 	}
+	o.githubClient, err = o.github.GitHubClient(nil, false)
```

Comment:
If you pass `nil` as the secret agent and the user _does_ provide a token this won't work

Reply:
Oups, that was an oversight. Fixed and tested that it works both with and without a token and does not work with an invalid token :)